collections/_article/communications-update-2023.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Prepare to hear a lot more from us!"
+excerpt: "The Godot Foundation is onboarding a person in charge of Community & Communications, and have a feedback form for you to fill out."
```

Comment:
Inconsistency between "is" and "have" - "is" suggests The Godot Foundation is singular, but "have" suggests it is plural. So should be either:

- "is" and "has", or
- "are" and "have"

I think!